### ISSUE TITLE
DAP minimum contrast issue for Table sort icon

### ISSUE THREAD

**Issue by polanders** [IBM/carbon-components-angular#181]
There is a minimum contrast issue with `ibm-table` where the sort icon in the header is hidden by setting `opacity: 0`.
The sort icon is hidden until the header gets focus or hover.

![image](https://user-images.githubusercontent.com/23030962/47153578-f0874a80-d2d7-11e8-93cb-23578f2d8026.png)

**Comment by zvonimirfras** [comment]
This looks like something that should be raised in carbon-components styles repo if it isn't already.
Possible solution is to use `display` instead of `opacity`, although that has some implications with space elements take even when are fully transparent.

Closing for now. Reopen if fix in styles requires coding changes.
